presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve performance of sum and average aggregations on decimals.
+* Disabled the Aggregation IF to FILTER rewrite rule by default. Updated the rule to stop unwrapping the IF expression instead Aggregation.
```

Comment:
@yuanzhanhku  why was this disabled? When should it be enabled then?

Reply:
It was disabled because we found a corner case which could return error if we apply this rewrite.
Instead of enabling it by default in the code, we will enable it in verifier/test clusters first and then enable it in production clusters with session properties.